app/assets/javascripts/listings/components/propertyCard.js.coffee

```diff
@@ -44,6 +44,23 @@ angular.module('dahlia.components')
             types.push(ListingDataService.reservedLabel(listing, type, 'reservedForWhoAre'))
         if types.length then types.join(' or ') else ''
 
+      @hasSaleUnitsWithoutParking = (listing) ->
```

Comment:
I'm wondering if these 3 functions are enough to warrant having components for sale and rental stats. thoughts?

Reply:
@akegan if you don't mind making them components then i would say go for it. if the partials weren't taking in variables i would be less inclined, but since we're also passing them vars, i would componentize.